### ISSUE TITLE
Improved home page refresh logic or manual refresh button

### ISSUE THREAD

**Issue by viggy96** [jellyfin/jellyfin-androidtv#1149]
**Describe the feature you'd like**	
Right now there's no way to trigger a refresh of the app home page, and it doesn't refresh often enough. The user could see a certain set of shows under "next up" on the Android TV home screen, but when the Jellyfin app is opened, the next up list is out of date, as are the other rows, latest shows, latest movies and so on. User is forced to force close the Jellyfin Android TV app, and re-open the app to get updated items on home page.

**Comment by mueslimak3r** [comment]
this looks similar to #1043  where there are contexts where it should refresh but doesn't

**Comment by Amateur-God** [comment]
Can also confirm, I'm having to soft restart my Chromecast with android TV whenever I add new media to my library as it doesn't show up on the home screen when the app is opened from sleep mode

**Comment by ngurganus** [comment]
I also am experiencing this issue.  New media added does not appear and progress made on movies or tv shows (from other clients) does not update.

**Comment by jumoog** [comment]
I open the app and see the old content. I open the app again and the content is updated.

**Comment by DenisBY** [comment]
still not implemented

**Comment by Rezer** [comment]
> still not implemented

What he said

**Comment by jprak-dotcom** [comment]
Yes please add a refresh button for the page in the app. In most apps you can scroll from up to down to trigger a refresh. A refresh button would also be fine.

**Comment by viggy96** [comment]
Unstale

**Comment by trettstadt** [comment]
It's still an issue.

**Comment by sorryusernameisalreadytaken** [comment]
I built a small test artifact here:
https://github.com/sorryusernameisalreadytaken/jellyfin-androidtv/actions/runs/34271028367

It should stay available for a few days....

I mainly built this to test a WebSocket reconnect issue when waking my google-tv-streamer-4K from suspend, but it also improved the home page refresh behavior for me.... So it could for you too

For libraries that already had existing media, newly added movies/shows started showing up in the home rows again without having to restart the app.

I think the separate case with completely new libraries, or rows that were empty before, might still not be covered by this sdk-fix alone.

If anyone wants to test it, just download the ZIP from the artifact, install the debug apk, and give it a try. :) It is a separate app (and ist slower than the normal stable build app because its a debug app).

I’d be interested to know whether this also improves the bug described here for you, or if it turns out to be unrelated in this case here.
